Dockerfile

```diff
@@ -1,5 +1,12 @@
-FROM golang:1.7
-ADD . /go/src/go.uber.org/yarpc
-RUN go install go.uber.org/yarpc/internal/crossdock
-ENTRYPOINT /go/bin/crossdock
+FROM golang:1.7.4
+
 EXPOSE 8080-8087
+RUN mkdir -p /go/src/go.uber.org/yarpc
+WORKDIR /go/src/go.uber.org/yarpc
+ADD glide.yaml /go/src/go.uber.org/yarpc/
+ADD glide.lock /go/src/go.uber.org/yarpc/
```

Comment:
Nit: This can be a single instruction

    ADD glide.yaml glide.lock /go/src/go.uber.org/yarpc/